src/DataTable/tests/FilterStatus.test.jsx

```diff
@@ -15,6 +15,17 @@ const filterProps = {
   onClick: () => {},
   clearFiltersText: 'CLEAR ME',
   className: 'filterClass',
+  showFilteredFields: true,
+};
+
+const filterPropsNoFiltered = {
```

Comment:
I think you can just copy out the prior filterProps using spread operator and only override the one value like this 

```
const filterPropsNoFiltered = {
    ...filterProps, 
    showFilteredFields: true,
}
```